modules/nw-pod-network-connectivity-check-object.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+//
+
+[id="nw-pod-network-connectivity-check-object_{context}"]
+= PodNetworkConnectivityCheck object fields
+
+The `PodNetworkConnectivityCheck` object fields are described in the following tables.
+
+.PodNetworkConnectivityCheck object fields
+[cols="2,1,3a",options="header"]
+|===
+
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the object in the following format: `<source>-to-<target>`. The destination described by `<target>` includes one of following strings:
+
+* `load-balancer-api-external`
+* `load-balancer-api-internal`
+* `kubernetes-apiserver-endpoint`
+* `kubernetes-apiserver-service-cluster`
+* `network-check-target`
+* `openshift-apiserver-endpoint`
+* `openshift-apiserver-service-cluster`
+
+|`metadata.namespace`
+|`string`
+|The namespace that the object is associated with. This value is always `openshift-network-diagnostics`.
+
+|`spec.sourcePod`
+|`string`
+|The name of the pod where the connection check originates, such as `network-check-source-596b4c6566-rgh92`.
+
+|`spec.targetEndpoint`
+|`string`
+|The target of the connection check, such as `api.devcluster.example.com:6443`.
+
+|`spec.tlsClientCert`
+|`object`
+|An object that configures the TLS certificate to use.
+
+|`spec.tlsClientCert.name`
+|`string`
+|The name of the TLS certificate used, if any. The default value is an empty string.
+
+|`status`
+|`object`
+|An object representing the condition of the connection test and logs of recent connection successes and failures.
+
+| `status.conditions`
+| `array`
+| An array populated with the last status of the connection check.
```

Comment:
Do we need the _An Array_? Maybe just _The last status of the connection check_?? And the other array fields also?

Reply:
@mburke5678, I'll try to omit _array_ because it's already disclosed in the type field for these.